test/jdk/com/sun/crypto/provider/KeyAgreement/DHKeyFactory.java

```diff
@@ -55,15 +58,13 @@ public static void main(String argv[]) throws Exception {
 
     private void run() throws Exception {
 
-        DHParameterSpec dhSkipParamSpec;
-
-        // use some pre-generated, default DH parameters
-        System.err.println("Using SKIP Diffie-Hellman parameters");
-        dhSkipParamSpec = new DHParameterSpec(skip1024Modulus,
-                                              skip1024Base);
+        jdk.test.lib.security.DiffieHellmanGroup dhGroup = jdk.test.lib.security.SecurityUtils.getTestDHGroup();
+        DHParameterSpec dhParamSpec;
+        System.err.println("Using " + dhGroup.name() + " Diffie-Hellman parameters");
+        dhParamSpec = new DHParameterSpec(dhGroup.getPrime(), dhGroup.getBase());
```

Comment:
Why not just
```
DHParameterSpec dhParamSpec = new DHParameterSpec(dhGroup.getPrime(), dhGroup.getBase());
System.err.println("Using " + dhGroup.name() + " Diffie-Hellman parameters");
```

I generally just put generic test output to `System.out` and messages related to failure/unexpected state to `System.err`.